src/internal/carriers/shipments/shipment-confirmation.ts

```diff
@@ -28,8 +28,10 @@ export class ShipmentConfirmation extends ShipmentIdentifierBase {
     return Boolean(this.trackingNumber);
   }
 
-  public get package(): PackageConfirmation {
-    return this.packages[0];
+  public get package(): PackageConfirmation | undefined {
+    if(this.packages && this.packages.length > 0) {
+      return this.packages[0];
+    }
```

Comment:
The `if` statement is unnecessary, since `this.packages` will always be an array.  All that's needed is `return this.packages[0];`